src/spinner/spinner.js

```diff
@@ -25,6 +25,15 @@ class Spinner extends React.Component<SpinnerPropsT> {
     overrides: {},
   };
 
+  componentDidMount() {
+    // TODO(v10): remove warning when switching default Spinner
+    console.warn(
```

Comment:
It should log only in a dev env.

Reply:
backfilled a few additional missing checks here: https://github.com/uber/baseweb/pull/2542